lib/chef/win32/security.rb

```diff
@@ -687,6 +687,12 @@ def self.test_and_raise_lsa_nt_status(result)
           Chef::ReservedNames::Win32::Error.raise!(nil, win32_error)
         end
       end
+
+      def self.error_insufficient_buffer?
```

Comment:
I'd prefer to just inline this.  It is creating an abstraction we don't really need.  The name is also confusing since NO_ERROR is not an insufficient buffer error.